frontend/src/metabase/components/Modal/WindowModal.tsx

```diff
@@ -77,6 +85,7 @@ export class WindowModal extends Component<WindowModalProps> {
           <div
             className={cx(className, CS.relative, CS.bgWhite, CS.rounded)}
             role="dialog"
+            data-element-id="modal"
```

Comment:
yeah, I think I would definitely prefer test ID here over anything else

Reply:
Fixed in https://github.com/metabase/metabase/pull/41091/commits/7a41e269ebcbed9cb7bfe3114a08cd3a3dc4a27c.

Although, it's worth noting that we're currently using `data-element-id` in some places already.